INSTALL.md

```diff
@@ -399,6 +408,11 @@ If you intend to run a server with translations, there are a few steps to follow
 
     This will download the .po files for your language(s) of choice to the po/
     folder with the correct filenames.
+    If you want to get /all/ translations, you can also use `tx pull -a`.
```

Comment:
asterisks will give actual italics in markdown
```suggestion
    If you want to get *all* translations, you can also use `tx pull -a`.
```

Reply:
I intended italics, but have no problem it switching to bold.

Reply:
I know :) I meant that `*all*` (or `_all_`) will display in italics as intended.

Reply:
My bad - thought asterisks was bold. Is also the disadvantage of making the edit in vi - no markdown preview :)